serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -14,11 +14,7 @@ Building Serverless Workflow Images using Quarkus CLI
 This document describes how to build Serverless Application Container Image using the link:{quarkus_cli_url}[Quarkus CLI].
 
 .Prerequisites
-* Ensure that the required dependencies are installed.
-* Your Serverless Workflow application is ready to be built. For more information about creating your Serverless Workflow application, see xref:getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service].
-* link:{quarkus_cli_url}[Quarkus CLI] {quarkus_version} is installed.
-* Java {java_min_version} is installed.
-* Apache Maven {maven_min_version} or later is installed.
+include::../../pages/_common-content/getting-started-requirement.adoc[]
```

Comment:
Sorry. I'm not sure I'm following you. Do you mean this could be a variable?

Reply:
Question: Are we planning to add more prereqs in the common file? 
If not, then we can write these sentences in the docs instead of using an `include`, since others might not be aware of this practice of using the common prereq file. 

wdyt?

Reply:
I don't think we'll add more prereqs in the common file.
However, when/if we do, we will have to change it in every guide if we don't include the prereqs file.
Also, having the prereqs file will be easier for us to keep the consistency.
We can include it in the new contributor's guide to make others aware of it.

WDYT @ricardozanini?

Reply:
I think it's good to have consistency between the guides, although now it's one sentence that can change in the future.

Also, @wmedvede suggested we might have a few more prerequisites for other scenarios such as Postgres, Kafka, and Kubernetes. We agreed to steer in this direction.